echo.go

```
@@ -13,6 +13,13 @@ func AddEchoRoutes(router *echo.Group) {
 
 func runTests(c echo.Context) error {
 	results := Run()
+
+	if instance.db != nil {
+		// TODO do we want to alert this somehow?
+		// doesn't make sense to return an http error since we do have test results
```

is it possible to add a suite-level log message?

---

Probably. Might not get noticed but we also don't actually use this endpoint currently since it's synchronous; it could use some attention at some point.

---

Probably wouldn't hurt to print errors other than `ErrNoDB` now and figure out making it better later. Silent errors can be annoying.

---

The problem is the lack of a logging library.